backend/app/transcendence/.eslintrc.js

```diff
@@ -0,0 +1,327 @@
+            "error",
+            "double"
+        ],
+        "semi":
+        [
+            "error",
+            "always"
+        ],
+        // custom rules based on 42
+        // POSSIBLE PROBLEMS
+        "no-await-in-loop": "error",
+        "no-class-assign": "error",
+        "no-compare-neg-zero": "error",
+        "no-cond-assign":
+        [
+            "error",
+            "always"
+        ],
+        "no-const-assign": "error",
+        "no-constant-condition": "error",
+        "no-constructor-return": "error",
+        "no-dupe-args": "error",
+        "no-dupe-class-members": "error",
+        "no-dupe-else-if": "error",
+        "no-dupe-keys": "error",
+        "no-duplicate-case": "error",
+        "no-duplicate-imports":
+        [
+            "error",
+            {
+                "includeExports": true
+            }
+        ],
+        "no-ex-assign": "error",
+        "no-fallthrough": 
+        [
+            "error",
+            {
+                "commentPattern": "break[\\s\\w]*omitted"
+            }
+        ],
+        "no-import-assign": "error",
+        "no-inner-declarations": "error",
+        "no-self-compare": "error",
+        "no-use-before-define": "error",
+        // SUGESTED
+        "accessor-pairs":
+        [
+            "error",
+            {
+                "getWithoutSet": true,
+                "setWithoutGet": true,
+                "enforceForClassMembers": false
+            }
+        ],
+        "arrow-body-style":
+        [
+            "error",
+            "always"
+        ],
+        "block-scoped-var": "error",
+        "camelcase":
+        [
+            "error",
+            {
+                "ignoreImports": true
+            }
+        ],
+        "consistent-return": "error",
+        "curly":
+        [
+            "error",
+            "multi"
+        ],
+        "default-case": "error",
+        "default-case-last": "error",
+        "dot-notation": "error",
+        "eqeqeq":
+        [
+            "error",
+            "smart"
+        ],
+        "func-style":
+        [
+            "error",
+            "declaration",
+            {
+                "allowArrowFunctions": true
+            }
+        ],
+        "grouped-accessor-pairs": "error",
+        "init-declarations":
+        [
+            "error",
+            "never",
+            {
+                "ignoreForLoopInit": true
+            }
+        ],
+        "max-classes-per-file": "error",
+        "max-depth":
+        [
+            "error",
+            4
+        ],
+        "max-lines-per-function":
+        [
+            "error",
+            {
+                "max": 25,
+                "skipComments": true,
+                "skipBlankLines": true
+            }
+        ],
+        "max-params":
+        [
+            "error",
+            4
+        ],
+        "max-statements":
+        [
+            "error",
+            4
+        ],
+        // "new-cap": "error",
```

Comment:
why is this commented? Does it serve a specific purpose?

Reply:
Yes testing purpose, like other no longer needed